src/software/sensor_fusion/refbox_data.h

```diff
@@ -0,0 +1,259 @@
+#pragma once
+
+#include <ostream>
+#include <vector>
+
+#include "software/new_geom/point.h"
+#include "software/proto/ssl_referee.pb.h"
+#include "software/util/time/timestamp.h"
+
+enum class RefboxGameState
+{
+    // these enum items map to the constants in RefboxCommand.msg
+    HALT                 = 0,
+    STOP                 = 1,
+    NORMAL_START         = 2,
+    FORCE_START          = 3,
+    PREPARE_KICKOFF_US   = 4,
+    PREPARE_KICKOFF_THEM = 5,
+    PREPARE_PENALTY_US   = 6,
+    PREPARE_PENALTY_THEM = 7,
+    DIRECT_FREE_US       = 8,
+    DIRECT_FREE_THEM     = 9,
+    INDIRECT_FREE_US     = 10,
+    INDIRECT_FREE_THEM   = 11,
+    TIMEOUT_US           = 12,
+    TIMEOUT_THEM         = 13,
+    GOAL_US              = 14,
+    GOAL_THEM            = 15,
+    BALL_PLACEMENT_US    = 16,
+    BALL_PLACEMENT_THEM  = 17,
+    REFBOX_GAME_STATE_COUNT
+};
```

Comment:
Is this duplicated from `refbox_constants.h`? Why?

Reply:
Yes it is. I forgot to delete refbox_constants.h and make everything use that. refbox_data.h should be the single source of info about internal refbox data structures

Reply:
fixed

Reply:
Unless I'm missing it somehow, doesn't look like `refbox_constants.h` is deleted in this PR?

Reply:
my bad, fixed now